HW2/RA_AdditionForLoop.java

```diff
@@ -0,0 +1,109 @@
+import java.util.Scanner;
+import java.util.Random;
+
+class RA_AdditionForLoop {
+   public static void main(String[] args) { 
+   
+   Scanner input = new Scanner(System.in);
+   Random rand = new Random(); 
+     
+     
+     
+  
+     int count = 0; 
+     int right = 0;
+     int wrong = 0; 
+     
+     
+     
+     
+     
+      
+        for (int i = 0; i < 10; i++) {
```

Comment:
we dont need a counter variable if you need to create a counter i, its the same thing, no need 2 variables